classes/controller/FrontController.php

```diff
@@ -1099,6 +1105,20 @@ public function unregisterJavascript($id)
         $this->javascriptManager->unregisterById($id);
     }
 
+    /**
+     * @param string $id
+     * @param string $relativePath
+     *
+     * @return string
+     */
+    private function buildRelativePath(string $id, string $relativePath)
+    {
+        $addSlash = substr($relativePath, 0, 1) !== '/' ? '/' : '';
+        $themeDir = substr($id, 0, 6) == 'theme-' ? _THEMES_DIR_ . _THEME_NAME_ : '';
```

Comment:
This is used to detect the compiled assets?

Reply:
No. Compiled assets has managed in CccReducer.php. $themeDir permits to define the real path to the asset in the theme (which is not given during the registration). Example :
```php
$this->registerStylesheet('theme-error', '/assets/css/error.css', ['media' => 'all', 'priority' => 50]);
```
The file is here : `themes/classic/assets/css/error.css`.

Reply:
Ooohh, so to include a theme assets you need to use an ID starting with `theme-wtf` ?
You should add a comment about that then, it's not very intuitive (I wonder if it's documented)

Reply:
I fixed it to a better way.